java/src/main/java/com/mlt/converter/encodings/GeometryEncoder.java

```diff
@@ -33,12 +34,15 @@ private GeometryEncoder() {}
   public static EncodedGeometryColumn encodeGeometryColumn(
       List<Geometry> geometries,
       PhysicalLevelTechnique physicalLevelTechnique,
-      SortSettings sortSettings) {
+      SortSettings sortSettings,
+      boolean triangulatePolygons) {
```

Comment:
Could we make it possible this take the triangulated polygons as an input to this function instead of embedding the triangulation inside of it?

On planetiler I'm considering some options for fixing invalid polygons that produce triangulated geometries as an intermediate step and I'd like for it to at least be possible to pass those triangulated polygons into this routine forcing them to be computed again.